base/lthooks.dtx

```diff
@@ -3911,17 +3921,18 @@
 %<latexrelease>  {
 %<latexrelease>    \@@_if_file_hook:wTF #1 / \s_@@_mark
 %<latexrelease>      {
-%<latexrelease>        \exp_args:Ne \@@_try_declaring_generic_hook_split:nNNnn
-%<latexrelease>          { \exp_args:Ne \@@_file_hook_normalize:n {#1} }
-%<latexrelease>      }
-%<latexrelease>      { \@@_try_declaring_generic_hook_split:nNNnn {#1} }
+%<latexrelease>\exp_args:Ne \@@_try_declaring_generic_hook_split:nNNnn
+%<latexrelease>  { \exp_args:Ne \@@_file_hook_normalize:n {#1} }
+%<latexrelease>     }
+%<latexrelease>     { \@@_try_declaring_generic_hook_split:nNNnn {#1} }
```

Comment:
Looks like the file had some tabs rather than real spaces, but that should be resolved to replacing them with the correct number of spaces, e.g via emacs `untabify` not by dropping them.

I consider correct and consistent indentation as important. I don't mind if the indentation increase is small (or differs from file to file) but withing a macro definition, please ensure that the indentation matches.
I guess it is mainly in the undocumented latexrelease blocks, but even there I think it is helpful 
for understanding the code.

There are a bunch of those below that I'm not highlighting individually.

Reply:
I may have been mistaken about the tabs, maybe there where just sequences of spaces, but they looked to my like tabls being replaced.